grafana/scylla-detailed.template.json

```diff
@@ -1815,6 +1815,21 @@
                         ],
                         "description": "scylla_lsa_non_lsa_used_space_bytes",
                         "title": "Non-LSA used memory"
+                    },
+                    {
+                        "class": "percentunit_panel",
+                        "span": 4,
+                        "targets": [
+                            {
+                                "expr": "$func(scylla_sstables_bloom_filter_memory_size{instance=~\"[[node]]\",cluster=\"$cluster\", dc=~\"$dc\", shard=~\"[[shard]]\"}) by ([[by]])/$func(scylla_memory_total_memory{instance=~\"[[node]]\",cluster=\"$cluster\", dc=~\"$dc\", shard=~\"[[shard]]\"}) by ([[by]])",
```

Comment:
This expression is going to be wrong if you choose `min` or `max` as the `$func` and there are shard with different amounts of memory in the cluster, because `max(scylla_sstables_bloom_filter_memory_size)` can come from a different shard than `max(scylla_memory_total_memory)`.

Are we fine with that?

Reply:
if we are talking about per-instance (which is what we care about) as long as the memory between shards on the same node is the same, it will be ok, I agree it's not perfect, but it is helpful when you have lots of shards and you are looking for the instance

Reply:
> if we are talking about per-instance (which is what we care about)

It will work for a per-instance view, but not for per-cluster or per-DC.

Reply:
I think it's useful per instance (or per shard). The bubble help can perhaps add a one liner 'accurate for per-node view only' or something.

Reply:
It will not work only if you are looking for a max over dc for example and the memory is not equal between shards, it's such a corner case, that I don't think it's important